docker/README.md

```diff
@@ -0,0 +1,38 @@
+# Cygnus Containerization
```

Comment:
Please, add a contact section at the end of the README. Just your name and email address.